### DISCUSSION TITLE
Low match rates

### DISCUSSION THREAD

**Discussion by labxYZY** [PGScatalog/pgsc_calc#350] [Q&A] [answered]
I'm using your tool to calculate PRS scores on UK Biobank but I'm finding that many of the matches such as PGS000010_hmPOS_GRCh37, PGS004861_hmPOS_GRCh37, PGS000010_hmPOS_GRCh37, etc. are well below 0.75, which is quite shocking, theoretically! UKB's genetic data is big enough that it can't cover the variants in these PGSs.
I'm sure they're all genotyped as GRCh37 versions.
I think this is a bit unusual and would like to ask if you have any thoughts on this? I do now QC the genotype data for maf, hwe etc and was wondering if I should remove those QCs.

As well as whether I can just leave out the min_overlap (set to 0?). I am not trying to compare PRS for individual diseases, I am developing further multi-PRS based on these SINGLE PRS, a before and after comparison, do you think it is feasible for me to just ignore min_overlap? Looking forward to your reply and thanks again for your great tools.

**Comment by smlmbrt** [comment]
> I think this is a bit unusual and would like to ask if you have any thoughts on this? I do now QC the genotype data for maf, hwe etc and was wondering if I should remove those QCs.

Some thoughts off the top of my head: do this scores have many ambiguous variants, these are dropped by default but can be added back if you're confident of strand orientation. It's also possible your QC thresholds are too strict?

> As well as whether I can just leave out the min_overlap (set to 0?). 

You can set this threshold to whatever you're comfortable with; however, it's important to note that they may not faithfully reproduce the original results of the PGS.

**Reply by labxYZY** [reply]
谢谢你的答复，我check了一下match log，发现有一些PGS的匹配率都在60~70之间，所以我也认为质量控制过于严格，打算更改一下质量控制限制。

**Comment by nebfield** [comment]
[The variant matching log](https://pgsc-calc.readthedocs.io/en/latest/explanation/output.html#match) should also be helpful to describe why variants aren't matching.

**Reply by labxYZY** [reply]
But on the other hand I found that some PGS obviously have hundreds of thousands of variants but the match rate is 0. This is a bit unbelievable, is it because they are all ambiguous and multi-allelic?
Or maybe it's the same problem as this one and I just need to change to version 2.0.0-beta.3? How do I specify which version to use?
![image](https://github.com/user-attachments/assets/f6e8b7e1-5111-4119-9ba8-5cb4fdaa41a8)

https://github.com/PGScatalog/pgsc_calc/issues/358

**Reply by smlmbrt** [reply]
@Links2000, if you could try the latest release ([`v2.0.0-beta.3`](https://github.com/PGScatalog/pgsc_calc/releases/tag/v2.0.0-beta.3)) and let us know if that fixes it? There was a problem with the log.

**Reply by labxYZY** [reply, marked as answer]
![image](https://github.com/user-attachments/assets/41425eac-9827-48a1-b8fb-de70cbc85bc8)
I replaced the v2.0.0-beta.3 version and this problem was solved, thank you very much.
